client/dashboard/sites/overview-plan-card/index.tsx

```diff
@@ -110,6 +110,7 @@ export default function PlanCard( { site }: { site: Site } ) {
 						progressValue={ progressBarValue }
 						progressColor={ storageWarningColor }
 						progressLabel={ `${ storageUsagePercent }%` }
+						isLoading={ isLoadingMediaStorage }
```

Comment:
To me, it's a bit off that we're passing `isLoading` to this `<Stat />` component, where we are passing a required `metric` and `description` (so that consumer needs to have fallback like you do with `?? 0` here), but we also render a placeholder when `isLoading` is true. Too much gymnastics here.

It seems a better way is to let `metric` and `description` to be undefined/null. When they are, then we render a placeholder, which can be a `<TextSkeleton />` (see existing example)

What do you think?

Reply:
> it's a bit off that we're passing isLoading to this `<Stat />` component, where we are passing a required `metric`

That's a good point. It's mean for the component to demand a `metric` when it clearly admits that there might not be one yet.

I think `isLoading` still needs to be explicit though because how could the component determine whether `description === undefined` because it is still loading or because this stat just doesn't want to have a description ever.

But I agree, I should use `<TextSkeleton>` and I will also make `metric` optional.